eng/common/scripts/Update-DocsMsPackages.ps1

```diff
@@ -18,10 +18,17 @@ packages which have not released to a central package manager.
 .PARAMETER DocRepoLocation
```

Comment:
Reminder this will need to be come from your tools PR.

Reply:
It's merging: https://github.com/Azure/azure-sdk-tools/pull/2035